drucker.proto

```diff
@@ -102,6 +102,13 @@ service DruckerWorker {
     rpc Predict_ArrString_ArrString (ArrStringInput) returns (ArrStringOutput) {}
 }
 
+enum EnumVersionInfo {
+    // Version info. The largest number is the latest version.
+    idx_2 = 0;
```

Comment:
Not a strong opposition, but why not simply using like `default = 2` ?

Reply:
Since this is grpc proto, `idx_2 = 0` does not mean `idx_2` is `0` but means the index of `idx_2` is `0`. Index `0` (value `idx_2`) will be a default value if we don't set `EnumVersionInfo`.

Reply:
I see. This seems reasonable to me now.
Should we change the naming `s/idx_2/latest_idx_2/` for better self-clarification?